src/components/Dashboard/FirmsAddressList.vue

```diff
@@ -0,0 +1,172 @@
+                <v-list-item-subtitle>
+                  <ul class="address-subtitle pre-line">
+                    <li class="address-line1">{{ businessAddress.deliveryAddress.streetAddress }}</li>
+                    <li class="address-line2">{{ businessAddress.deliveryAddress.streetAddressAdditional }}</li>
+                    <li class="address-line3">
+                      {{ businessAddress.deliveryAddress.addressCity }}
+                      {{ businessAddress.deliveryAddress.addressRegion }}
+                      {{ businessAddress.deliveryAddress.postalCode }}
+                    </li>
+                    <li class="address-line4">
+                      <span>{{ getCountryName(businessAddress.deliveryAddress.addressCountry) }}</span>
+                    </li>
+                  </ul>
+                </v-list-item-subtitle>
+              </v-list-item-content>
+            </v-list-item>
+
+            <v-list-item class="mailing-address-list-item" v-if="businessAddress.mailingAddress">
+              <v-list-item-icon class="address-icon mr-0">
+                <v-icon color="primary">mdi-email-outline</v-icon>
+              </v-list-item-icon>
+              <v-list-item-content>
+                <v-list-item-title class="mb-2 address-title">Mailing Address</v-list-item-title>
+                <v-list-item-subtitle>
+                  <div
+                    class="same-as-above"
+                    v-if="isSame(businessAddress.deliveryAddress, businessAddress.mailingAddress, ['id'])">
+                    <span>Same as above</span>
+                  </div>
+                  <ul v-else class="address-subtitle pre-line">
+                    <li class="address-line1">{{ businessAddress.mailingAddress.streetAddress }}</li>
+                    <li class="address-line2">{{ businessAddress.mailingAddress.streetAddressAdditional }}</li>
+                    <li class="address-line3">
+                      {{ businessAddress.mailingAddress.addressCity }}
+                      {{ businessAddress.mailingAddress.addressRegion }}
+                      {{ businessAddress.mailingAddress.postalCode }}
+                    </li>
+                    <li class="address-line4">
+                      <span>{{ getCountryName(businessAddress.mailingAddress.addressCountry) }}</span>
+                    </li>
+                  </ul>
+                </v-list-item-subtitle>
+              </v-list-item-content>
+            </v-list-item>
+          </v-list>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Mixins, Prop } from 'vue-property-decorator'
+import { State } from 'vuex-class'
+
+// Mixins
+import { CommonMixin, CountriesProvincesMixin } from '@/mixins'
+
+// Interfaces
+import { OfficeAddressIF } from '@/interfaces'
+
+@Component({})
+export default class FirmsAddressList extends Mixins(CommonMixin, CountriesProvincesMixin) {
```

Comment:
This all looks substantially similar to the existing template/script in AddressListSm.vue. Couldn't you blend the changes into that instead of duplicating so much code?

Reply:
OK, Lekshmi and I chatted and determined it's too messy to do this all in AddressListSm, so I'm OK with this sub-component.